### PR TITLE
Added --path and warnings before execution

### PR DESCRIPTION
**Pull request by fuerbringer**
:rocket:

### CODE REVIEW

File: charreplace.go [hunk 1]
```diff
@@ -10,22 +10,48 @@
...
 
+
+	if(len(os.Args) <= 1) {
+		var confirm string
+		dir, _ := os.Getwd()
+		fmt.Printf("Warning: This action will possibly corrupt files in the directory %s", dir)
+		fmt.Printf("Continue [y,n] ")
+		fmt.Scanln(&confirm)
+		if(strings.ToLower(confirm) != "y") {
+			fmt.Printf("Aborting!")
```
Comment (Hoi15A) [inline charreplace.go]: It would be nice if this were a `Println` instead of `Printf` as it looks slightly better in the console.

**Review by Hoi15A — CHANGES_REQUESTED** [review]
Passing any flag thats not '--path' will result in instant run of the replacement, that does not seem intended and should be adressed. Also it would be nice if a shorthand -p were an option.

I also left a comment on an output to console that can be improved.

**Comment by fuerbringer** [comment]
Quality feedback! I fixed all the issues.

**Comment by Hoi15A** [comment]
Seems to work pretty well, I have one minor question though. When using a flag that is not known, `--wtf` for example, it just runs as if there is no flag. Thats better than before but I'd normally expect it to throw an error instead, a bit like how `tldr` does for example:
```
austin@austin-neon:~/Documents/git/PRs$ tldr --wtf

  error: unknown option `--wtf'

austin@austin-neon:\~/Documents/git/PRs$ ./charreplacer-go/charreplace --wtf ./
Warning: This action will possibly corrupt files in the directory /home/austin/Documents/git/PRs
Continue? [y,n] n
Aborting!
```

I'm not 100% sure if its standard though, but I feel like that would be a good thing to have for general consistency.

**Comment by fuerbringer** [comment]
@Hoi15A Unsure about wether or not it is a standard. But the CLI tools I use certainly seem to catch unknown or invalid flags. It makes sense so I added a small check for that.

**Comment by Hoi15A** [comment]
I dont see anything wrong now, so ill merge this pull. Well done :) :+1: :rocket:

**Pull request merged**